src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CaseDataControllerTest.java

```diff
@@ -215,6 +215,7 @@ public void shouldSuccessfullyReturnNotAsAdminContestedPaperCase() throws Except
     public void shouldSuccessfullySetOrgPolicy() throws Exception {
         when(idamService.isUserRoleAdmin(isA(String.class))).thenReturn(false);
         when(caseDataService.isContestedApplication(any())).thenReturn(true);
+        when(caseDataService.isConsentedApplication(any())).thenReturn(true);
```

Comment:
you are mocking your test to return true for both contested and consented? are you sure about this?

Reply:
Thanks Mike, removed my entry for the test.